WordPress/src/main/java/org/wordpress/android/ui/stories/StoryComposerActivity.kt

```diff
@@ -202,6 +236,22 @@ class StoryComposerActivity : ComposeLoopFrameActivity(),
         )
     }
 
+    private fun saveInitialPost() {
+        editPostRepository.set {
+            val post: PostModel = postStore.instantiatePostModel(site, false, null, null)
+            post.setStatus(DRAFT.toString())
+            post.setPostFormat(POST_FORMAT_WP_STORY_KEY)
+            post
+        }
+        editPostRepository.savePostSnapshot()
+        // this is an artifact to be able to call savePostToDb()
+        // also, Story posts are always PUBLISHED
+        editPostRepository.getEditablePost()?.setStatus(PUBLISHED.toString())
+        site?.let {
+            savePostToDbUseCase.savePostToDb(WordPress.getContext(), editPostRepository, it)
```

Comment:
Same static context comment 🙏

Reply:
did in https://github.com/wordpress-mobile/WordPress-Android/commit/86fb2ad01d5fba438d6c20979d5c6c03f36e2ceb

Reply:
this one addressed in fb65ea2d46